packages/tizen_package_manager/README.md

```diff
@@ -0,0 +1,83 @@
+# tizen_package_manager
+
+ [![pub package](https://img.shields.io/pub/v/tizen_package_manager.svg)](https://pub.dev/packages/tizen_package_manager)
+
+Tizen package manager API. Used for getting installed package info.
+
+## Usage
+
+To use this package, add `tizen_package_manager` as a dependency in your `pubspec.yaml` file.
+
+```yaml
+dependencies:
+  tizen_package_manager: ^0.1.0
+```
+
+### Retrieving specific package info
+
+To retrieve information for the specific package,  get `PackageInfo` with `getPackageInfo` method.
```

Comment:
> By the way, wouldn't it be better to follow the type/method names used by [TizenFX](https://docs.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.PackageManager.html)? For example,

It would be better to keep current method/type name for consistency of [tizen_app_manager](https://github.com/flutter-tizen/plugins/pull/325)